tests/e2e/util.go

```diff
@@ -3786,7 +3792,7 @@ func toggleCSIMigrationFeatureGatesOnkublet(ctx context.Context,
 	sshClientConfig := &ssh.ClientConfig{
 		User: "root",
 		Auth: []ssh.AuthMethod{
-			ssh.Password("ca$hc0w"),
+			ssh.Password(esxPassword),
```

Comment:
Declare a new var for k8sMasterVMPassword and use it here

Reply:
Changed to use a different variable